package-lock.json

```diff
@@ -14,6 +14,16 @@
         "stylelint-selector-bem-pattern": "^3.0.1"
       }
     },
+    "node_modules/@aashutoshrathi/word-wrap": {
```

Comment:
Do we need these changes in the package-lock.json? I dont see any changes to package.json?

Reply:
I could just delete the lock file if that works @anthonyhashemi

Reply:
just need to revert the changes in it, not delete it